releases/release-4.0.9.md

```diff
@@ -0,0 +1,168 @@
+---
+title: TiDB 4.0.9 Release Notes
+---
+
+# TiDB 4.0.9 Release Notes
+
+Release date: December 10, 2020
+
+TiDB version: 4.0.9
+
+## Compatibility changes
+
++ TiDB
+
+    - Deprecate `enable-streaming` [#21055](https://github.com/pingcap/tidb/pull/21055)
+
+## New Features
+
++ TiFlash
+
+    - Support storing the latest data of the storage engine on multiple disks
```

Comment:
Finally, this is an experimental feature in the release-4.0 branch. Should we add some tips in the release note? Like adding `(experimental)` at the end of the line. @TomShawn

Reply:
Of course.
```suggestion
    - Support storing the latest data of the storage engine on multiple disks (experimental)
```